onnxruntime/__init__.py

```diff
@@ -9,7 +9,10 @@
 __version__ = "1.2.0"
 __author__ = "Microsoft"
 
-from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, RunOptions, SessionOptions, set_default_logger_severity, NodeArg, ModelMetadata, GraphOptimizationLevel, ExecutionMode
-from onnxruntime.capi.session import InferenceSession
+from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, set_seed, RunOptions, SessionOptions, set_default_logger_severity, NodeArg, ModelMetadata, GraphOptimizationLevel, ExecutionMode, OrtDevice, SessionIOBinding
+from onnxruntime.capi.session import InferenceSession, IOBinding
 from onnxruntime.capi import onnxruntime_validation
+
+from onnxruntime.capi.training import *
+
```

Comment:
If someone builds with training disabled could this import fail?

Reply:
there's another placeholder version used in that case: [https://github.com/microsoft/onnxruntime/blob/d92906bae930d3326c5e9e2348b99b97f03ad469/cmake/onnxruntime_python.cmake#L169](https://github.com/microsoft/onnxruntime/blob/d92906bae930d3326c5e9e2348b99b97f03ad469/cmake/onnxruntime_python.cmake#L169)

---
In reply to: [414366007](https://github.com/microsoft/onnxruntime/pull/3681#discussion_r414366007) [](ancestors = 414366007)